src/LiteCore/src/LiteCore.Shared/Interop/Fleece.cs

```diff
@@ -549,6 +551,9 @@ public static void FLEncode(this object obj, FLEncoder* enc)
                 case Blob b:
                     b.FLEncode(enc);
                     break;
+                case SecureString ss:
+                    new NetworkCredential(string.Empty, ss).Password.FLEncode(enc);
```

Comment:
The NetworkCredential class internally always stores the password as a SecureString:
https://referencesource.microsoft.com/#System/net/System/Net/NetworkCredential.cs,125

Reply:
This ends up in a string in memory anyway so the question is: "Does it really provide more security than before?"